requirements.txt

```diff
@@ -0,0 +1,4 @@
+testtools
```

Comment:
Please refer to the setup.py file for the packages that need to be added to this file.